pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "wisvch-dienst2"
+version = "1.13"
```

Comment:
Where does this version come from? I personally don't think we should do any versioning.